core/src/main/java/com/sequenceiq/cloudbreak/structuredevent/rest/StructuredEventFilter.java

```diff
@@ -86,11 +88,17 @@
 
     private static final int MAX_CONTENT_LENGTH = 65535;
 
+    private static final String ID = "id";
+
+    private static final String CRN = "crn";
+
     private final List<String> skippedHeadersList = Lists.newArrayList("authorization");
 
     private final Map<String, WorkspaceResourceRepository<?, ?>> pathRepositoryMap = new HashMap<>();
 
-    private final Pattern extendRestParamsFromResponsePattern = Pattern.compile("\"id\":([0-9]*)");
+    private final Pattern extendIdRestParamFromResponsePattern = Pattern.compile("\"" + ID + "\":([0-9]*)");
+
+    private final Pattern extendCrnRestParamFromResponsePattern = Pattern.compile("\"" + CRN + "\":\"([0-9a-zA-Z:-]*)\"");
```

Comment:
What is extending crn rest param from response exactly? I don't understand what this pattern does.

Reply:
check the tests then :) when the response is not json. I don't know when it can happen

Reply:
then call it extracting instead of extending please, because with this name it does not make sense